morseCode/validateInput.js

```diff
@@ -0,0 +1,23 @@
+const { wordsToMCResult } = require('./wordsToCodeMorse');
+const { mCTWordsResult } = require('./morseCodeToWords');
+const { InputError } = require('../utils/errors');
+const { letters } = require('../utils/variables');
+
+
+const inputValidation = (userInput) => {
```

Comment:
This is not only doing input validation.
It's also calling the appropriate function to solve the problem.

I think the function is good, just the name shouldn't be `inputValidation` in my opinion.

Reply:
You're right so, I changed the output of `inputValidation()`  to true or false.
This way I can only test the input.

Reply:
```suggestion
const inputValidation = (input) => {
  let validation;
  const REGEX = /[.-]/gm;
  if (input.search(REGEX) >= 0) {
    if (input.search(LETTERS) >= 0) {
      throw new InputError('Insert a valid Morse Code. No letter or numbers allowed.');
    } else {
      validation = true;
      return validation;
    }
  } else {
    validation = false
    return validation;
  }
};
```